CMakeLists.txt

```diff
@@ -167,8 +165,8 @@ if(ASAN)
 endif(ASAN)
 
 if(SHADOW)
+  set(WITH_STATIC OFF)
   set(WITH_SHARED ON)
-  add_compile_options(-fPIC)
```

Comment:
why is win32 messing with Shadow?

Reply:
`-[fK][Pp][Ii][Cc]` is set globally due to abseil requiring it